### PR TITLE
Add dataset URI to Task from CLI

### LINKED ISSUE #2976: Add dataset URI to Task from CLI
### My actions before raising this issue
- [ x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [ x] Searched [past issues](/issues)

### Expected Behaviour

It would be useful to programmatically attach a  git repository for annotation synchronization. 

### Current Behaviour

Either cannot be done or the CLI is missing documentation on how to do it.

### Next steps
You may [join our Gitter](https://gitter.im/opencv-cvat/public) channel for community support.

### PR DESCRIPTION
**Pull request by dreaquil**
### Motivation and context

It would be useful to programmatically attach a git repository for annotation synchronization.

Closes #2976.

### How has this been tested?
Tested this in a personal deployment of CVAT over HTTPS.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: utils/cli/core/core.py [hunk 1]
```diff
@@ -97,6 +98,16 @@ def tasks_create(self, name, labels, overlap, segment_size, bug, resource_type,
                 log.info(logger_string)
 
             self.tasks_upload(task_id, annotation_format, annotation_path, **kwargs)
+        if dataset_repository_url:
+            git_url = self.api.git
```
Comment (azhavoro) [inline utils/cli/core/core.py:102]: @dreaquil When trying to create a task, I got the following error
```python
Traceback (most recent call last):
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 42, in <module>
    main()
  File "/home/andrey/workspace/cvat/utils/cli/cli.py", line 34, in main
    actions[args.action](cli, **args.__dict__)
  File "/home/andrey/workspace/cvat/utils/cli/core/core.py", line 102, in tasks_create
    git_url = self.api.git
AttributeError: 'CVAT_API_V1' object has no attribute 'git'
```

Comment (dreaquil) [reply utils/cli/core/core.py:102]: Apologies.. sems to not have pushed. Pushed now.

File: utils/cli/core/core.py [hunk 2]
```diff
@@ -97,6 +98,16 @@ def tasks_create(self, name, labels, overlap, segment_size, bug, resource_type,
...
+            response = self.session.post(
+                        f'{git_url}create/{task_id}',
+                        json={
+                            'path': dataset_repository_url,
+                            'lfs': lfs,
+                            'tid': task_id})
+            response_json = response.json()
+            log.info(f"Create: {response_json.rq_id}")
```
Comment (azhavoro) [inline utils/cli/core/core.py:110]: Please check the response `status_code` here, the request may fail.
Also, seems the line should be changed `response_json.rq_id -> response_json['rq_id']`

Additional comment: I think it is necessary to check the status of add of the remote repo with `{git_url}/check_process` to make sure everything is ok, or the print error message

**Comment by azhavoro** [comment]
@dreaquil Thanks for the contribution, please fix minor comment about checking request status.

File: utils/cli/core/core.py (hunk 2 above)
Comment (dreaquil) [reply utils/cli/core/core.py]: Updated. Please confirm that this is satisfactory.

**Review by azhavoro — APPROVED** [review]

**Pull request merged**